arduino/examples/ESP32WiFiClientBridge/ESP32WiFiClientBridge.ino

```diff
@@ -105,7 +105,8 @@ void setup() {
     openMRNServer.setNoDelay(true);
     openMRNServer.begin();
     MDNS.begin(hostname);
-    MDNS.addService(openlcb::TcpDefs::MDNS_SERVICE_NAME_GRIDCONNECT_CAN, "tcp", OPENMRN_TCP_PORT);
+    MDNS.addService(openlcb::TcpDefs::MDNS_SERVICE_NAME_GRIDCONNECT_CAN,
+        openlcb::TcpDefs::MDNS_PROTOCOL_TCP, OPENMRN_TCP_PORT);
```

Comment:
previously this was "tcp" and now it is "_tcp". Is that going to be a problem?

Reply:
no issues, the mDNS system prepends the _ only if missing